src/main/java/application/models/MultipleChoiceQuestion.java

```diff
@@ -1,9 +1,142 @@
 package application.models;
 
 import javax.persistence.DiscriminatorValue;
+import javax.persistence.ElementCollection;
 import javax.persistence.Entity;
-
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+/**
+ * Subclass of Question class
+ * Create a multiple choice question
+ */
 @Entity
 @DiscriminatorValue("1")
 public class MultipleChoiceQuestion extends Question{
+
+    /**
+     *  Holds the collection of choices a user can choose from
+     */
+    @ElementCollection
+     private Collection<String> choices;
```

Comment:
Would this end up storing the choices twice?

Reply:
Yes it does, once in a list and then in a map as values. I may be wrong, but this seems to be getting a bit convuluted, when a list of choices and a list of answers would be sufficient for now.

Reply:
My concern is that if there isn't an id mapping available, it will ends up passing only strings to controller/view. Subsequently it might ends up having browser sending strings when it could have been an id, which could be a problem scalability wise (although we probably don't care). But yeah I guess for prototype it is fine.

Reply:
Yeah, I could remove it but when we get to making the radio buttons for the choices were going to provide some collection of choices. If needed we can remove it later and rework the code to iterate through the map to get them.